tuned.spec

```diff
@@ -518,7 +526,11 @@ fi
 %{_prefix}/lib/tuned/spindown-disk
 %{_mandir}/man7/tuned-profiles-compat.7*
 
-%changelog
```

Comment:
I think you didn't mean to remove the changelog :).

Reply:
crap! Of course not :)